### PR TITLE
fix corner case of plethysm with tensors

### PR DESCRIPTION
**Pull request by zabrocki**
There was a corner case of the plethysm function that was failing when computing a plethysm of an empty partition and a tensor of symmetric functions.

This change inserts the base case of a product in the plethsym formula as a default of 1 to the .one() of the ring.

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [x] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

**Comment by zabrocki** [comment]
@anneschilling @tscrim @mantepse Fixing a corner case.

File: src/sage/combinat/sf/sfa.py [hunk 1]
```diff
@@ -3601,6 +3601,8 @@ def plethysm(self, x, include=None, exclude=None):
             sage: s = SymmetricFunctions(T).s()
             sage: s[2](5)
             15*B[] # B[]
+            sage: s[[]](tensor([p[1], s[1]]))
+            p[] # s[]
```
Comment (anneschilling) [inline src/sage/combinat/sf/sfa.py]:
```suggestion

        Fixed :issue:`41257`::

            sage: s[[]](tensor([p[1], s[1]]))
            p[] # s[]
```

**Review by anneschilling — CHANGES_REQUESTED** [review]

**Review by anneschilling — APPROVED** [review]
Looks good! Wait for patchbot to check and then set positive review.

File: src/sage/combinat/sf/sfa.py [hunk 2]
```diff
@@ -3646,11 +3651,11 @@ def plethysm(self, x, include=None, exclude=None):
         if tensorflag:
             tparents = Px._sets
             lincomb = Px.linear_combination
-            elt = lincomb((prod(lincomb((tensor([p[r].plethysm(base(la))
+            elt = lincomb((prod((lincomb((tensor([p[r].plethysm(base(la))
                                                  for base, la in zip(tparents, trm)]),
                                          _raise_variables(c, r, degree_one))
                                         for trm, c in x)
-                                for r in mu),
+                                for r in mu),tensor([base.one() for base in tparents])),
```
Comment (mantepse) [inline src/sage/combinat/sf/sfa.py]:
```suggestion
                                for r in mu), tensor([base.one() for base in tparents])),
```
